src/llvm/LLVMDG2Dot.h

```diff
@@ -95,6 +95,20 @@ class LLVMDG2Dot : public debug::DG2Dot<LLVMNode>
             os << "\\nERR: no BB";
         }
 
+        //Print Location in source file
+        if (const llvm::Instruction *I = llvm::dyn_cast<llvm::Instruction>(val))
+        {
```

Comment:
Please keep the coding style of the project and put this '{' on the same line as the if